docs/api.html

```diff
@@ -48,14 +49,14 @@
 	The Consumer API allows applications to read streams of data from topics in the Kafka cluster.
 	<p>
 	Examples showing how to use the consumer are given in the
-	<a href="/{{version}}/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html" title="Kafka 0.10.2 Javadoc">javadocs</a>.
+	<a href="/{{version}}/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html" title="Kafka {{version}} Javadoc">javadocs</a>.
```

Comment:
@ewencp, @guozhangwang Can you please confirm if it's OK to use `{{version}}` in the title too? Just want to confirm if it was intentional to hardcode the version number in the title.